src/test/java/com/google/sps/SchedulerTest.java

```diff
@@ -297,4 +325,49 @@ public void nonRequiredWithLabBothIncluded() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  public List<Course> createTrash() {
```

Comment:
Nit: Do you still wanna commit the test to `SchedulerTest.java`? If so, maybe change name lol :)

Reply:
haha thanks